.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+# Related Issue
+- fixes #[ISSUE-NUMBER]
+
+# Summary
+
+# Proposed Changes
+- change 1
+- change 2
+
+# Screenshots (if appropriate)
+
+Original                |         Updated
+:------------------------:|:------------------------:
+** original screenshot ** | ** updated screenshot ** |
+
+# Bugs introduced by this patch
+- bug 1
+- bug 2
+
+## Other information
```

Comment:
This template is already in another previous PR that is almost ready to merge. No need to duplicate it here and cause conflicts to resolve. Please remove this file.

Reply:
Apologies for not taking a proper look at the other pull request, I am going to remove this file too. Thank you for taking out the time to review.